lib/slice.ark

```diff
@@ -0,0 +1,16 @@
+{
+  (let slice (fun (string escapeCharacter){
+    #made by Natendrtfm <3
```

Comment:
better putting the comment at the top of file, and adding a space after the `#`
Also you can add the date and specify that Natendrtfm is your github username like so: `Made with <3 by @Natendrtfm (on Github)`